src/modules/standard-clauses/standard-clauses.controller.ts

```
@@ -1,62 +1,119 @@
-  @ApiResponse({ status: 200, description: 'The standard clause has been successfully updated.', type: StandardClause })
+  @ApiResponse({
+    status: 200,
+    description: 'The standard clause has been successfully updated.',
+    type: StandardClause,
+  })
   @ApiResponse({ status: 404, description: 'Standard clause not found.' })
   update(
     @Param('id') id: string,
-    @Body() updateStandardClauseDto: Partial<CreateStandardClauseDto>,
+    @Body() updateStandardClauseDto: UpdateStandardClauseDto,
+    @Body() updateStandardClauseDto: UpdateStandardClauseDto,
```

Biome (1.9.4)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)